__test__/Student.spec.js

```diff
@@ -0,0 +1,44 @@
+const {registerPayload, loginPayload, deletePayload} =  require("./Student_test_data");
+const axios = require("axios");
+
+//These tests will work locally, but not in CicleCI because we do not have a public API available.
+//Change the URL when that is setup.
+test("add a new Student ", async () => {
+  
+  let httpResponse = await axios.post('http://localhost:5000/Student/add',registerPayload)
```

Comment:
Do not use axios here ... import the route directly and use it.
see here  : https://fek.io/blog/how-to-add-unit-testing-to-express-using-jest